pilot/pkg/bootstrap/server.go

```diff
@@ -17,14 +17,22 @@ package bootstrap
 import (
 	"fmt"
 	"net"
+	"net/http"
 	"os"
 	"path"
+	"strconv"
 	"time"
 
+	"google.golang.org/grpc"
+
```

Comment:
imports follow one style, run/fmt.sh

Reply:
Well - run/fmt.sh seems to be broken, doesn't match linter...